tests/nonsmoke/functional/CompileTests/OpenMP_tests/targetupdate.c

```diff
@@ -0,0 +1,14 @@
+void main(float*p, float*v1, float*v2, int N)
```

Comment:
The main function of a C program has two particular parameters: `int` and `char**`, which are the number of arguments and what they are. It returns an `int`.
This doesn't affect the correctness of unparsing, but it would be better to follow the tradition.
Next time, please check all your test cases and rename the function to another name, like `foo` or some other meaningful names (`axpy`, `matmul`, etc`) depending on your kernel.